hackernews-ingest/bulk_ingest/bulk_ingest.py

```diff
@@ -118,6 +125,7 @@ def deserialize_to_dict(item):
             tag_set=tags,
             num_value=row.get("score", 0),
             tracking_id=str(row.get("id")),
+            upsert_by_tracking_id=True,
```

Comment:
Upsert is now true, since bulk insert is able to properly handle bulk upserts efficiently